website/docs/d/data_lake_pipeline_runs.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: data_lake_pipeline_runs"
+sidebar_current: "docs-mongodbatlas-datasource-data-lake-pipeline-runs"
+description: |-
+    Describes Data Lake Pipeline Runs.
+---
+
+# Data Source: mongodbatlas_data_lake_pipeline_runs
+
+`mongodbatlas_data_lake_pipeline_run` describe Data Lake Pipeline Runs.
+
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```terraform
+data "mongodbatlas_data_lake_pipeline_runs" "test" {
```

Comment:
Often times in our examples we have shown setting up the resource that would create the object this data source is getting information about.  I've included an example above but they've been very helpful for users.

Reply:
Updated